internal/server/v1/optimus/crud.go

```diff
@@ -0,0 +1,24 @@
+package optimus
+
+import (
+	"net/http"
+
+	optimusv1beta1 "buf.build/gen/go/gotocompany/proton/grpc/go/gotocompany/optimus/core/v1beta1/corev1beta1grpc"
+	"github.com/go-chi/chi/v5"
+
+	"github.com/goto/dex/internal/server/utils"
+)
+
+func handleGetOptimus(client optimusv1beta1.JobSpecificationServiceClient) http.HandlerFunc {
```

Comment:
we can separate service `service.go` with http handler `handler.go`.

handler scopes:
1. interacting with request object e.g. getting values from route params and querystring
2. building response object. e.g. deciding what status code to return
3. get current user (if required)

Service scopes:
1. interacting with optimus grpc
2. business logics (if any)